vllm/model_executor/model_loader.py

```diff
@@ -62,7 +63,17 @@ def get_model(model_config: ModelConfig) -> nn.Module:
         # Create a model instance.
         # The weights will be initialized as empty tensors.
         with torch.device("cuda"):
-            model = model_class(model_config.hf_config, linear_method)
+            if getattr(model_class, "supports_lora", True):
```

Comment:
I believe this should use default `False` :  `getattr(model_class, "supports_lora", False)`